cpu/stm32_common/periph/rtc.c

```diff
@@ -71,6 +71,10 @@
 #define EXTI_PR_BIT         (EXTI_PR_PR17)
 #endif
 
+#if defined(CPU_MODEL_STM32L412KB)
+    #define RTC_ISR_ALRAF RTC_SR_ALRAF
```

Comment:
Although I prefer this indentation, it is not the way it is done for the rest of the code, so please change this to:

```
#if defined(CPU_MODEL_STM32L412KB)
#define RTC_ISR_ALRAF RTC_SR_ALRAF
#endif
```

Reply:
I think
```C
#ifndef RTC_ISR_ALRAF 
#define RTC_ISR_ALRAF RTC_SR_ALRAF
#endif 
```

would be a more generic solution.